src/store/apiSlice.js

```diff
@@ -7,7 +7,12 @@ export const apiSlice = createApi({
     getSources: builder.query({
       query: () => '/provisioning/v1/sources',
     }),
+    getArchitecturesByDistribution: builder.query({
+      query: (distribution) =>
+        `/image-builder/v1/architectures/${distribution}`,
```

Comment:
This is absolutely correct, but just want to give a warning that in #953 I changed the baseUrl from '/api' to '' so this will need to become '/api/image-builder/...' if #953 gets merged or if you rebase onto #953.